cmd/telemetrygen/README.md

```diff
@@ -0,0 +1,31 @@
+# Telemetry generator for OpenTelemetry
+
+| Status                   |                       |
+| ------------------------ | --------------------- |
+| Stability                | traces [WIP]          |
+|                          | metrics [WIP]         |
+| Supported pipeline types | traces, metrics       |
+| Distributions            | [contrib]             |
```

Comment:
Since this is not a component (like e.g. the Datadog exporter) we should not put anything on the `Distributions` field 

```suggestion
| Supported signal types | traces, metrics       |
| Distributions            | N/A         |
```

or alternatively

```suggestion
| Supported signal types | traces, metrics       |
```